lib/hyacinth/adapters/external_identifier_adapter/datacite/rest_api/v2/api.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+require 'json'
+class Hyacinth::Adapters::ExternalIdentifierAdapter::Datacite::RestApi::V2::Api
+  MINIMUM_REQUIRED_ATTRIBUTES_FOR_FINDABLE_DOI = [:title,
+                                                  :creators,
+                                                  :doi_url,
+                                                  :publisher,
+                                                  :publication_year,
+                                                  :resource_type,
+                                                  :doi_prefix].freeze
+  # Events used when minting a DOI, depends on desired state of minted DOI
+  # see https://support.datacite.org/docs/how-do-i-make-a-findable-doi-with-the-rest-api
+  DOI_STATES = [:draft, :findable, :registered].freeze
+  DOI_MINT_EVENT = { draft: '',
+                     findable: 'publish',
+                     registered: 'hide' }.freeze
+
+  attr_accessor :most_recent_request_body,
+                :most_recent_response
```

Comment:
I see that these are read from in a couple of places, but they don't appear to be written to so they'll always be nil.  I'm guessing you deleted some things because of Ben's comment about thread safety, and meant to clean this up too?

Reply:
accessors have been removed.